readme.md

```diff
@@ -0,0 +1,65 @@
+# tesr.finance 
+
+## Local development setup
+
+- Clone the [repository](git@github.com:MVPWorkshop/tesseract-app.git) in to your local environment
+  ```
+  git clone git@github.com:<username>/tesseract-app.git
+  ```
+- Create a `.env` file in the root directory
+  ```
+  cp .env.example .env
+  ```
+
+  These are the variables we use for local development
+
+  ```
+  REACT_APP_API_BASE_URL=https://prom.tesr.finance/api/v1
```

Comment:
We can use the actual .env.example file, makes cloning the project easier and easier to maintain than to have to update both the README and the .env.example file.

If we ever have values which shouldn't be "public", we just leave that place blank and add it specifically to the README on how to obtain

Reply:
got it. Will add these in the example instead.